R/generate.R

```diff
@@ -72,30 +72,35 @@ random_matrix <- function(objects) {
   random_data_class <- sample(matrix_classes, 1)
   sample_data <- function(data) { sample(data, random_width * random_height, replace = TRUE) }
   random_data <- switch(random_data_class,
-    integer       = sample_data(c(objects$negative_integers, objects$positive_integers)),
-    double        = sample_data(c(objects$negative_doubles, objects$positive_doubles)),
-    logical       = sample_data(objects$logicals),
-    character     = sample_data(c(objects$characters, objects$utf8)),
-    simple_string = sample_data(random_simple_strings(random_width * random_height,
-      objects = objects)))
+                        integer       = sample_data(c(objects$negative_integers, objects$positive_integers)),
+                        double        = sample_data(c(objects$negative_doubles, objects$positive_doubles)),
+                        logical       = sample_data(objects$logicals),
+                        character     = sample_data(c(objects$characters, objects$utf8)),
+                        simple_string = sample_data(random_simple_strings(random_width * random_height,
+                                                                          objects = objects)))
   matrix(random_data, random_width, random_height)
 }
 
 #' Get all the user-installed dataframes through data()
+#' @importFrom utils data
```

Comment:
Lets add `utils` to the `Imports` in the `DESCRIPTION` file too.

Reply:
(I think if we do that, we won't need to prefix with `utils::` anymore, but being explicit is fine.)